pibooth/booth.py

```diff
@@ -307,6 +307,8 @@ def find_capture_event(self, events):
         for event in events:
             if event.type == pygame.KEYDOWN and event.key == pygame.K_p:
                 return event
+            if event.type == pygame.KEYDOWN and event.key == pygame.K_LEFT:
+                return event
```

Comment:
I think we can merge this: 
```suggestion
            if event.type == pygame.KEYDOWN and event.key in [pygame.K_p, pygame.K_LEFT]:
                return event
```